sdlib/d/gc/tcache.d

```diff
@@ -347,6 +377,29 @@ private:
 		return true;
 	}
 
+	void freeImpl(PageDescriptor pd, void* ptr) {
+		pd.arena.free(emap, pd, ptr);
```

Comment:
I'm not sure this is such a great idea, as it hides the fact this is using the emap. But the emap need to be initialized before it is used.